test/connectors/SubscriptionFieldsConnectorSpec.scala

```diff
@@ -330,22 +331,19 @@ class SubscriptionFieldsConnectorSpec extends UnitSpec with ScalaFutures with Mo
   "saveFieldValues" should {
 
     val fieldsValues = fields("field001" -> "value001", "field002" -> "value002")
-    val subFieldsPutRequest = SubscriptionFieldsPutRequest(clientId, apiContext, apiVersion, fieldsValues)
-
-    val putUrl = s"$urlPrefix/application/$clientId/context/$apiContext/version/$apiVersion"
 
     "save the fields" in new Setup {
-      when(mockHttpClient.PUT[SubscriptionFieldsPutRequest, HttpResponse](any(), any())(any(),any(),any(),any()))
+      when(mockHttpClient.PUT[SubscriptionFieldsPutRequest, HttpResponse](any(), any(), any())(any(), any(), any(), any()))
         .thenReturn(Future.successful(HttpResponse(OK)))
 
       await(subscriptionFieldsConnector.saveFieldValues(clientId, apiContext, apiVersion, fieldsValues))
 
-      verify(mockHttpClient).PUT[SubscriptionFieldsPutRequest, HttpResponse](meq(putUrl), meq(subFieldsPutRequest))(any(),any(),any(),any())
+      verify(mockHttpClient).PUT[SubscriptionFieldsPutRequest, HttpResponse](any(), any(), any())(any(), any(), any(), any())
```

Comment:
Should this verify check the first two arguments (I'm not sure what the 3rd argument is?)

Reply:
The 3rd argument (headers) comes from upgrading to a newer version of http-verbs.
It was using: https://github.com/hmrc/http-verbs/blob/e73e64310485050aa5aba62d33c62b4f5480fef3/src/main/scala/uk/gov/hmrc/http/HttpPut.scala#L28
It's now using: https://github.com/hmrc/http-verbs/blob/0d172fb72b95bc7036fa0a8b02c4fddb72275b9e/src/main/scala/uk/gov/hmrc/http/HttpPut.scala#L28

Reply:
We've also put the eqTo(putUrl), eqTo(subFieldsPutRequest) back in
`      verify(mockHttpClient).PUT[SubscriptionFieldsPutRequest, HttpResponse](eqTo(putUrl), eqTo(subFieldsPutRequest), any())(any(), any(), any(), any())
`